pkg/monitortestlibrary/pathologicaleventlibrary/duplicated_event_patterns.go

```diff
@@ -774,6 +771,42 @@ func IsDuringAPIServerProgressingOnSNO(topology string, events monitorapi.Interv
 	}
 }
 
+func addAllowedSingleNodeMatchers(registry *AllowedPathologicalEventRegistry, finalIntervals monitorapi.Intervals) {
+	snoTopology := v1.SingleReplicaTopologyMode
+
+	// This is a list of known and allowed single node events
+	allowedEvents := []EventMatcher{
+		&SimplePathologicalEventMatcher{
+			name:               "AllowSingleNodeOperatorControllerBackOffRestartingFailedContainer",
+			messageReasonRegex: AllowBackOffRestartingFailedContainer.messageReasonRegex,
+			messageHumanRegex:  AllowBackOffRestartingFailedContainer.messageHumanRegex,
+			topology:           &snoTopology,
+			locatorKeyRegexes: map[monitorapi.LocatorKey]*regexp.Regexp{
+				monitorapi.LocatorNamespaceKey: regexp.MustCompile("openshift-operator-controller"),
+				monitorapi.LocatorPodKey:       regexp.MustCompile("operator-controller-controller-manager"),
+			},
+			jira: "https://issues.redhat.com/browse/OCPBUGS-45071",
+		},
+		&SimplePathologicalEventMatcher{
+			name:               "AllowSingleNodeCatalogdControllerBackOffRestartingFailedContainer",
+			messageReasonRegex: AllowBackOffRestartingFailedContainer.messageReasonRegex,
+			messageHumanRegex:  AllowBackOffRestartingFailedContainer.messageHumanRegex,
+			topology:           &snoTopology,
+			locatorKeyRegexes: map[monitorapi.LocatorKey]*regexp.Regexp{
+				monitorapi.LocatorNamespaceKey: regexp.MustCompile("openshift-catalogd"),
+				monitorapi.LocatorPodKey:       regexp.MustCompile("catalogd-controller-manager"),
+			},
+			jira: "https://issues.redhat.com/browse/OCPBUGS-45071",
+		},
```

Comment:
I think you may have over corrected :smile: .  Originally you were [filtering events](https://github.com/openshift/origin/blob/545ab25818987a00ff1938efbc9c89583f88b423/pkg/monitortests/node/legacynodemonitortests/pathological_events.go#L62) for `[sig-cluster-lifecycle] pathological event should not see excessive Back-off restarting failed containers`.  I don't think this does that anymore.

Reply:
The path to [testBackoffStartingFailedContainer](https://github.com/jeff-roche/origin/blob/678dab7de111723253c5f67a05ded69317cd29c7/pkg/monitortests/node/legacynodemonitortests/pathological_events.go#L59) is part of [monitortests/node/legacynodemonitortests](https://github.com/jeff-roche/origin/blob/678dab7de111723253c5f67a05ded69317cd29c7/pkg/monitortests/node/legacynodemonitortests/monitortest.go#L70) chain that starts with [monitor Stop](https://github.com/jeff-roche/origin/blob/678dab7de111723253c5f67a05ded69317cd29c7/pkg/monitor/monitor.go#L123)

But I believe you just moved the matchers under [testframework/legacytestframeworkmonitortests](https://github.com/jeff-roche/origin/blob/678dab7de111723253c5f67a05ded69317cd29c7/pkg/monitortests/testframework/legacytestframeworkmonitortests/monitortest.go#L56).  I don't think the filter gets applied to the code above (could be wrong)